flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -568,6 +571,32 @@ public InputDependencyConstraint getDefaultInputDependencyConstraint() {
 		return defaultInputDependencyConstraint;
 	}
 
+	/**
+	 * Enables to put all vertices into the same slot sharing group by default.
+	 */
+	@PublicEvolving
```

Comment:
Would it make sense to mark these methods as `@Internal`? We don't really expect users to build on top of these, do we?

Reply:
That makes sense, since it's mainly for blink batch. 
Let's keep it @Internal until it is required by users sometime in the future.

Reply:
done.